squiRL/common/agents.py

```diff
@@ -66,8 +61,8 @@ def get_action(
         action_logit = net(obs)
         probs = F.softmax(action_logit, dim=-1)
         dist = torch.distributions.Categorical(probs)
-        action = dist.sample().item()
-        action = int(action)
+        action = dist.sample().squeeze()
+        action = action.item() if action.ndim == 0 else action.numpy()
```

Comment:
In play_step you expand this into [action] when this returns an item. Would it be faster if you return action.expand_dims directly?

Reply:
Good catch! I didn't know about `np.expand_dims()`. Done and thanks!